### PR TITLE
add precurs

### PR DESCRIPTION
**Pull request by CRR257**

### CODE REVIEW

**Review by carlosclatg — CHANGES_REQUESTED** [review]
Please include README.md

File: staff/clara-rufi/Precurs2019/BingoHTML/Bingo.js [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+var columna9 = [] 
+
+fila1 = []
+fila2 = []
+fila3 = []
+
+nameUser = []
+
+function login(){
+
```
Comment (carlosclatg) [inline staff/clara-rufi/Precurs2019/BingoHTML/Bingo.js:28]: It is necessary to indent code.

File: staff/clara-rufi/Precurs2019/BingoHTML/Bingo.js [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+            fila3.push(number2)
+            document.getElementsByClassName("b")[k].value += number2 
+        }
+}
+
+//columna3
+for (var k=0; k<3; k++){
+    var number3 = (Math.floor(Math.random ()*(31-21))+21)
+        if (k===0){
+        columna3.push(number3)
```
Comment (carlosclatg) [inline staff/clara-rufi/Precurs2019/BingoHTML/Bingo.js:82]: Indent code in different levels.

File: staff/clara-rufi/Precurs2019/BingoHTML/Bingo.js [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+    }else if (columna1.indexOf(number1) === -1){
+        columna1.push(number1)
+        document.getElementsByClassName("a")[k].value += number1
+        fila3.push(number1) 
+  }
+}
+
+// columna2
+for (var k=0; k<3; k++){
+    var number2 = (Math.floor(Math.random ()*(21-11))+11); //Retorna un entero aleatorio entre min (incluido) y max (excluido)
```
Comment (carlosclatg) [inline staff/clara-rufi/Precurs2019/BingoHTML/Bingo.js:60]: set as constants, not as numbers. (21,11...) in everyrow

File: staff/clara-rufi/Precurs2019/BingoHTML/Bingo.js [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+}
+
+document.getElementById('requestlikeCarton').style="display:block;"
+document.getElementById('questionCarton').style="display:block;"
+document.getElementById("requestlikeCarton").value =  nameUser + ", te gusta este cartón o quieres uno de nuevo?"
+document.getElementById('Yes').style="display:block;"
+document.getElementById('No').style="display:block;"
+}
+
+function No(){
```
Comment (carlosclatg) [inline staff/clara-rufi/Precurs2019/BingoHTML/Bingo.js:217]: Name more specific.

File: staff/clara-rufi/Precurs2019/BingoHTML/Bingo.js (hunk 1 above)
Comment (CRR257) [reply staff/clara-rufi/Precurs2019/BingoHTML/Bingo.js:28]: ok, thanks for the improvement

File: staff/clara-rufi/Precurs2019/Calculadora_gat/app/app.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+function calculate(){
+    var n1=parseInt(document.getElementById("n1").value)
+    var n2=parseInt(document.getElementById("n2").value)
+    var sum = n1 + n2
+    
+    if (sum<50){
+    document.getElementById("results").innerHTML = "<li>" + n1 +  " + " + n2 + " = " + sum + "</li>" + "<p>" + "<li>" + "Today you have eaten " + sum + " cookies" + "</li>"
```
Comment (carlosclatg) [inline staff/clara-rufi/Precurs2019/Calculadora_gat/app/app.js:7]: indent

File: staff/clara-rufi/Precurs2019/Tema_1__Functions/Projecte_1.js [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+    resultRest = Math.round((num1 - num2)*1000)/1000;
+    resultMult = Math.round((num1 * num2)*1000)/1000;
+    resultDiv = "Indeterminado"
+    console.log('La suma de ' + num1 + ' mas ' + num2 + ' es igual a ' + resultSum);
+	console.log('La resta de ' + num1 + ' menos ' + num2 + ' es igual a ' + resultRest);
+    console.log('La multiplicación de ' + num1 + ' por ' + num2 + ' es igual a ' + resultMult);
+    console.log('La división de ' + num1 + ' por ' + num2 + ' es ' + resultDiv);
+    
+}else if (typeof num1=== "number" && typeof num2 === "number"){
+    resultSum = Math.round((num1 + num2)*1000)/1000;
```
Comment (carlosclatg) [inline staff/clara-rufi/Precurs2019/Tema_1__Functions/Projecte_1.js:94]: Is case n/0 take into account?

File: staff/clara-rufi/Precurs2019/Tema_3__Objectes/Projecte_3.js [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+    alert("Cartón Bingo => " + cartro) 
+    console.log("Cartón Bingo => " + cartro)
+    }
+    board();
+
+    var numberRandom = []
+	countX = 0
+    function numberBingo(){
+    var numberB = (Math.ceil(Math.random ()*15))
+    for (var F=0; F<30; F++){
```
Comment (carlosclatg) [inline staff/clara-rufi/Precurs2019/Tema_3__Objectes/Projecte_3.js:170]: variables in lower case.

**Pull request merged**